pkg/virt-operator/resource/apply/generations.go

```diff
@@ -3,19 +3,15 @@ package apply
 import (
 	"fmt"
 
-	"k8s.io/apimachinery/pkg/runtime"
-
-	k6tv1 "kubevirt.io/api/core/v1"
-
-	appsv1 "k8s.io/api/apps/v1"
-
-	operatorsv1 "github.com/openshift/api/operator/v1"
-	"github.com/openshift/library-go/pkg/operator/resource/resourcemerge"
 	admissionregistrationv1 "k8s.io/api/admissionregistration/v1"
+	appsv1 "k8s.io/api/apps/v1"
 	policyv1 "k8s.io/api/policy/v1"
 	extv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	k6tv1 "kubevirt.io/api/core/v1"
```

Comment:
Should be `v1` while `k8s.io/apimachinery/pkg/apis/meta/v1` is aliased to `metav1`. Just a side note, but you started touching the imports.

Reply:
I've reverted changes to the imports. It's not part of  this PR.